tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -291,3 +288,211 @@ def test_delete_during_rgw_svc_restart(self):
+        self.log.info("STARTED: Verify copy object after rgw restart using pkill")
+        object_name = 'object-test-42253'
+        bucket_list = list()
+        self.log.info("Step 1: Start write Operations :")
+        for i in range(0, 2):
+            bucket_name = f"bucket-test-42253-{i}"
+            resp = self.s3_test_obj.create_bucket(bucket_name=bucket_name)
+            assert_utils.assert_true(resp[0], resp[1])
+            bucket_list.append(bucket_name)
+        for size in self.test_cfg["size_list"]:
+            file_name = "{}{}".format("dtm-test-42253", size)
+            file_path = os.path.join(self.test_dir_path, file_name)
+            system_utils.create_file(file_path, size)
+            resp = self.s3_test_obj.put_object(bucket_list[0], f"{object_name}_{size}", file_path)
+            assert_utils.assert_true(resp[0], resp[1])
+        self.log.info("Step 2: Perform Single rgw Process Restart")
+        resp = self.dtm_obj.process_restart(master_node=self.master_node_list[0],
+                                            health_obj=self.health_obj,
+                                            pod_prefix=const.SERVER_POD_NAME_PREFIX,
+                                            container_prefix=const.RGW_CONTAINER_NAME,
+                                            process=self.rgw_process, check_proc_state=False)
+        assert_utils.assert_true(resp, "Failure observed during process restart")
+        self.log.info("Step 3: Perform Copy Object to bucket-2, download and verify on copied "
+                      "Objects")
+        for size in self.test_cfg["size_list"]:
+            resp = self.s3_test_obj.copy_object(source_bucket=bucket_list[0],
+                                                source_object=f"{object_name}_{size}",
+                                                dest_bucket=bucket_list[1],
+                                                dest_object=f"{object_name}_{size}")
+            assert_utils.assert_true(resp[0], resp[1])
+            file_name_copy = "{}{}".format("dtm-test-42253-copy", size)
+            file_path_copy = os.path.join(self.test_dir_path, file_name_copy)
+            resp = self.s3_test_obj.object_download(bucket_name=bucket_list[1],
+                                                    obj_name=f"{object_name}_{size}",
+                                                    file_path=file_path_copy)
+            assert_utils.assert_true(resp[0], resp[1])
+            file_name = "{}{}".format("dtm-test-42253-", size)
+            file_path = os.path.join(self.test_dir_path, file_name)
+            resp = system_utils.validate_checksum(file_path_1=file_path, file_path_2=file_path_copy)
+            assert_utils.assert_true(resp, "Checksum validation Failed.")
+        self.test_completed = True
+        self.log.info("ENDED: Verify copy object after rgw restart using pkill")
+
+    @pytest.mark.lc
+    @pytest.mark.dtm
```

Comment:
just realized that we do not have marker entry added to pytest.ini for dtm. Please add.

Reply:
@priyankaborawake is there any specific reason for not adding marker yet?

Reply:
@ApurwaMohite Will add this as part of my PR.